cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -400,11 +539,54 @@ def get_rna_bigwig_coverage_upload_summary(
             )
         return upload_summary
 
+    @staticmethod
+    def get_rna_fraser_outrider_upload_summary(
+        rna_dna_collections: list[RNADNACollection],
+    ) -> list[str]:
+        upload_summary: list[str] = []
+        for rna_dna_collection in rna_dna_collections:
+            upload_summary.extend(
+                f"Uploaded fraser and outrider files for sample {rna_dna_collection.dna_sample_name} in case {dna_case}."
+                for dna_case in rna_dna_collection.dna_case_ids
+            )
+        return upload_summary
+
+    @staticmethod
+    def get_rna_genome_build_upload_summary(
+        rna_dna_collections: list[RNADNACollection],
+    ) -> list[str]:
+        upload_summary: list[str] = []
+        for rna_dna_collection in rna_dna_collections:
+            upload_summary.extend(
+                f"Uploaded RNA genome build file for sample {rna_dna_collection.dna_sample_name} in case {dna_case}."
+                for dna_case in rna_dna_collection.dna_case_ids
+            )
+        return upload_summary
+
+    @staticmethod
+    def get_variant_load_summary(
+        rna_dna_collections: list[RNADNACollection],
+    ) -> list[str]:
+        upload_summary: list[str] = []
+        for rna_dna_collection in rna_dna_collections:
+            upload_summary.extend(
+                f"Loaded variant outlier for sample {rna_dna_collection.dna_sample_name} in case {dna_case}."
+                for dna_case in rna_dna_collection.dna_case_ids
+            )
+        return upload_summary
+
     def upload_rna_junctions_to_scout(self, dry_run: bool, case_id: str) -> None:
         """Upload RNA junctions splice files to Scout."""
         self.upload_splice_junctions_bed_to_scout(dry_run=dry_run, case_id=case_id)
         self.upload_rna_coverage_bigwig_to_scout(case_id=case_id, dry_run=dry_run)
 
+    def upload_omics_to_scout(self, dry_run: bool, case_id: str) -> None:
+        """Upload RNA omics files to Scout."""
+        self.upload_omics_sample_id_to_scout(dry_run=dry_run, case_id=case_id)
+        self.upload_rna_fraser_outrider_to_scout(dry_run=dry_run, case_id=case_id)
+        self.upload_rna_genome_build_to_scout(dry_run=dry_run, case_id=case_id)
+        self.load_rna_variant_outlier_to_scout(dry_run=dry_run, case_id=case_id)
```

Comment:
All these functions start the same way. Get the "collection" and then iterate through it.

Maybe we could fetch the collection here and then pass it to each of these methods?

Reply:
each of the cli command needs get the collection as they have to work separate from each other in the current logic

Reply:
I don't quite understand. Separate in what ways?

Reply:
I misinterpreted your comment, applied your comment